README.md

```diff
@@ -8,3 +8,5 @@ This is a portfolio of both my code and my actual website in which projects will
 
 ### Bin monitor
 This application was created during my second year of university and involved my first interaction with react.js. For this project I created a Arduino which would monitor a bins capacity and then be able to display this data using a web app, the idea being a waste management team would know which bins where full and even calculate which bins would need to be emptied more than once a day.
+##### Video
+[![Video](https://imgur.com/p866LxH.png)](https://www.youtube.com/watch?v=p7mcFBUQZxM&feature=youtu.be&fbclid=IwAR1ApEGBtzbGqX2Ir4tYi82M4p_RD10IvU8e83FFdJLxKVUxqS-yHcKUXZw "Video")
```

Comment:
Good use of external programs, try and re-record the video and upload onto your own channel incase the group delete it later on.